flake.nix

```diff
@@ -2,19 +2,14 @@
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
     flake-utils.url = "github:numtide/flake-utils";
-    fedimint = {
-      url = "github:fedimint/fedimint?rev=bbe587845935517df8a36b9b2a6b98fa4bc1e19e";
-    };
+    fedimint = { url = "github:fedimint/fedimint?branch=releases/0.2"; };
```

Comment:
Not entirely sure of nix's lock file behavior but this seems like it'll either be harder to update than using a hash or git tag, or that users may unexpectedly end up with `flake.lock` changes if someone pushes to releases/0.2. Any reason you don't want to stick with the current system of using hashes?

Reply:
You have to explicitly update the lockfile, it won't auto-update. Then it will use the most up-to-date version.

Reply:
Hmm, this also makes updating the README / docker compose a little harder since now it's not just a trivial find & replace. We had instructions over here which no longer work with this: https://github.com/fedimint/ui#bumping-referenced-fedimint

Reply:
kk I can update to a hash, do we want to cut a 0.2-rc4 in line with fedimint? @elsirion is there any change to the gateway rpc from your lightning module PR?

Reply:
Release candidates don't track (e.g. ui doesn't have an rc-N to match a fedimint rc-N) so no need to cut an rc4 if there are no changes necessary. Updating the dockerfiles / flake.nix doesn't have any affect on the build artifacts that are attached to tags.

Reply:
I verified that the UI works with rc5.